matchmaking.py

```diff
@@ -47,13 +47,23 @@ def create_challenge(self, username, base_time, increment, days, variant):
         except Exception:
             return None
 
+    def get_time(self, name, default=None):
+        match_time = self.matchmaking_cfg.get(name, default)
+        if match_time is None:
+            return

```

Comment:
`return None` is better here, since this function is expected to return a value instead of just exiting. I know `return` and `return None` are equivalent, but the second reads better in this context.